enterprise/backend/test/metabase_enterprise/advanced_config/file/databases_test.clj

```diff
@@ -59,24 +62,11 @@
     ;; make sure we're actually testing something if it was already set to false locally.
     (mt/with-temporary-setting-values [config-from-file-sync-databases true]
       (try
-        (binding [advanced-config.file/*supported-versions* {:min 1, :max 1}
-                  advanced-config.file/*config*             {:version 1
-                                                             :config
-                                                             ;; `settings:` HAS to come before `databases:`, otherwise the
-                                                             ;; flag won't be set when database sync stuff happens.
-                                                             ;;
-                                                             ;; Using a [[flatland.ordered.map]] here really isn't necessary
-                                                             ;; since this map only has two keys and will be created as an
-                                                             ;; `ArrayMap`, preserving the originally specified order... but
-                                                             ;; using [[ordered-map]] explicitly here makes this constraint
-                                                             ;; clearer I think. Also the YAML library reads stuff in as an
-                                                             ;; ordered map so this more closely matches the behavior when
-                                                             ;; using a file
-                                                             (ordered-map/ordered-map
-                                                              :settings  {:config-from-file-sync-databases false}
-                                                              :databases [{:name    test-db-name
-                                                                           :engine  "h2"
-                                                                           :details (:details (mt/db))}])}]
+        (binding [advanced-config.file/*config* {:version 1
+                                                 :config  {:settings  {:config-from-file-sync-databases false}
+                                                           :databases [{:name    test-db-name
+                                                                        :engine  "h2"
+                                                                        :details (:details (mt/db))}]}}]
```

Comment:
`:settings` doesn't have to come first anymore -- fixed this a few PRs ago -- but forgot to remove this comment